zuul.d/junipernetworks-junos-jobs.yaml

```diff
@@ -4,49 +4,17 @@
 - job:
     name: ansible-network-junos-appliance
     parent: ansible-network-appliance-base
-    pre-run: playbooks/ansible-network-junos-appliance/pre.yaml
+    pre-run:
+      - playbooks/ansible-network-junos-appliance/pre.yaml
+      - playbooks/network-ee-integration-tests/pre.yaml
     run: playbooks/ansible-network-junos-appliance/run.yaml
```

Comment:
this run needs to be removed, please refer to vyos-vyos-jobs.yaml for example

Reply:
actually, I am wrong... and tired :)

Reply:
yeah, I see it should remain. :)